pymatgen/io/vasp/inputs.py

```diff
@@ -428,7 +427,10 @@ def from_str(cls, data, default_names=None, read_velocities=True):
             crd_scale = scale if cart else 1
             coords.append([float(j) * crd_scale for j in tokens[:3]])
             if has_selective_dynamics:
-                selective_dynamics.append([tok.upper()[0] == "T" for tok in tokens[3:6]])
+                if all(value in {'T', 'F'} for value in tokens[3:6]):
+                    selective_dynamics.append([value == 'T' for value in tokens[3:6]])
```

Comment:
Please help me confirm that VASP indeed only takes the `token[3:6]` as selective dynamics values.